docker-storage-setup.sh

```diff
@@ -931,7 +928,7 @@ create_lv() {
 return 0
 }
 
-setup_docker_root_volume() {
+setup_root_volume() {
   if [ -z "$DOCKER_ROOT_VOLUME_SIZE" ]; then
```

Comment:
I intentionally left this, so that @rhatdan patch can take care of this.
https://github.com/projectatomic/docker-storage-setup/pull/181#issuecomment-272227206
Point (b)

However I can change it if you like ?

Shishir